Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -369,6 +370,32 @@ def get_detections(last_behavior_time=None, behavior_id=None, filter_arg=None):
     return response
 
 
+def get_fetch_detections(last_created_timestamp=None, behavior_id=None, filter_arg=None):
```

Comment:
we are not using the behavior_id, is it really needed?

Reply:
done